lib/bots/versionbot.ts

```diff
@@ -833,7 +866,10 @@ export class VersionBot extends ProcBot {
             moddedFiles.push(`CHANGELOG.md`);
 
             // Now we get the new version from the CHANGELOG (*not* the package.json, it may not exist).
-            return exec(`cat ${versionData.fullPath}${_.last(moddedFiles)}`).then((contents: string) => {
+            return cliCommand({
+                command: `cat`,
```

Comment:
It's really not a dumb question, and actually I was convinced that I'd changed this in another PR. Having spent time just now looking, I can't find the change. I'll make it here.